jni/patches/cadical-shared.patch

```diff
@@ -1,31 +1,34 @@
 diff --git a/configure b/configure
-index e761d26..43320d2 100755
+index c02b197..ba0c99b 100755
 --- a/configure
 +++ b/configure
-@@ -471,7 +471,11 @@ mobical:
+@@ -474,6 +474,12 @@ mobical:
  	\$(MAKE) -C "\$(CADICALBUILD)" mobical
  update:
  	\$(MAKE) -C "\$(CADICALBUILD)" update
--.PHONY: all cadical clean mobical test update
 +shared:
 +	\$(MAKE) -C "\$(CADICALBUILD)" libcadical.so
++dynamiclib:
++	\$(MAKE) -C "\$(CADICALBUILD)" libcadical.dylib
 +dll:
 +	\$(MAKE) -C "\$(CADICALBUILD)" cadical.dll
-+.PHONY: all cadical clean mobical test update shared dll
+ .PHONY: all cadical clean mobical test update
```

Comment:
Add `shared`, `dylib`, `dll` to .PHONY target.
Why you removed it?

Reply:
No, I forgot to add them because I made a patch from scratch instead of manually changing